src/bin/cargo/commands/remove.rs

```diff
@@ -66,7 +66,7 @@ pub fn exec(gctx: &mut GlobalContext, args: &ArgMatches) -> CliResult {
         print_available_packages(&workspace)?;
     }
 
-    let packages = args.packages_from_flags()?;
+    let packages = args.packages_from_flags(gctx, false)?;
```

Comment:
> set ignore_package as true to indicate that dont pop-up warnning, these commands needs this

why does `remove` not need this?

Reply:
`add` and `remove`  command don't support the --workspace option.